--- configure.orig	2024-02-28 09:44:26.148331951 +0100
+++ configure	2024-02-28 09:46:48.748513956 +0100
@@ -24855,19 +24855,19 @@
   LINUX_BUILD_FALSE=
 fi
 
-   if test x$with_cgroup = xyes; then
-  WITH_CGROUP_TRUE=
-  WITH_CGROUP_FALSE='#'
-else
+  #  if test x$with_cgroup = xyes; then
+  # WITH_CGROUP_TRUE=
+  # WITH_CGROUP_FALSE='#'
+# else
   WITH_CGROUP_TRUE='#'
   WITH_CGROUP_FALSE=
-fi
+# fi
 
-  if test x$with_cgroup = xyes; then
+#   if test x$with_cgroup = xyes; then
 
-printf "%s\n" "#define WITH_CGROUP 1" >>confdefs.h
+# printf "%s\n" "#define WITH_CGROUP 1" >>confdefs.h
 
-  fi
+#   fi
 
 
 
